shell/platform/embedder/embedder.h

```diff
@@ -369,6 +375,14 @@ typedef struct {
   uint32_t name;
   /// The texture format (example GL_RGBA8).
   uint32_t format;
+  /// The pixel data buffer.
+  const uint8_t* buffer;
+  /// The size of pixel buffer.
+  size_t buffer_size;
+  /// Callback invoked that the gpu surface texture start binding.
+  BoolCallback bind_callback;
+  /// The type of the texture.
+  FlutterGLImpellerTextureType impeller_texture_type;
   /// User data to be returned on the invocation of the destruction callback.
   void* user_data;
```

Comment:
Another opinion... How about using this `user_data`?

[engine commit ](https://github.com/JSUYA/engine/commit/e8c370a20ac2d69407a9ae9f876c542d7a3e9125) , [embedder commit](https://github.com/JSUYA/embedder/commit/86155eb3563763fed38bd3bc739b8c3681c571b4)

I declared embedder_tizen.h and reverted embedder.h to its original state. [[my embedder_tizen.h]](https://github.com/JSUYA/engine/commit/e8c370a20ac2d69407a9ae9f876c542d7a3e9125#diff-0b30f4ff79516408a59b9602ca84f1c43e6709529bfe313ba99122f655e61b33R1)

The existing code used user_data as ExternalTextureSurfaceEGLImpeller*.
But what I suggest is to use user_data as the new structure(FlutterOpenGLTextureTizen) we added.

embedder_external_texture_gl_impeller.cc can use data stored in user_data(`FlutterOpenGLTextureTizen`).
https://github.com/JSUYA/engine/commit/e8c370a20ac2d69407a9ae9f876c542d7a3e9125#diff-c8025734c594d1f506c30f2ae8e84c630c8270626874f93ba88c816fe3d85682
Then we can avoid changing the existing embedder.h.

+The `FlutterOpenGLTextureTizen` should be changed to an appropriate name.

Reply:
If our goal is to minimize mdifications to the flutter engine code and that code is only for Tizen Platofrm . I think it's a good idea.